src/viam/cmake/viamcppsdk_utils.cmake

```diff
@@ -2,7 +2,7 @@ function(viamcppsdk_add_boost_test SOURCE_FILE_NAME)
   get_filename_component(TEST_EXECUTABLE_NAME ${SOURCE_FILE_NAME} NAME_WE)
 
   add_executable(${TEST_EXECUTABLE_NAME} ${SOURCE_FILE_NAME})
-  target_link_libraries(${TEST_EXECUTABLE_NAME} viamcpp)
+  target_link_libraries(${TEST_EXECUTABLE_NAME} viam-cpp-sdk::viamsdk)
```

Comment:
The SDK library is now named `libviamsdk` rather than `libviamcppsdk`, and I've also gone ahead and started consistently accessing it through its alias name, as consumers would.